pages/api/admin/WriteID.js

```diff
@@ -0,0 +1,108 @@
+import { validateFunc } from "../validate";
+import { google } from "googleapis";
+
+import {service_info} from "../../../utils/decrypt.js";
+import firebase from '../../../firebase/clientApp'
+
+/*
+ * /api/admin/WriteID
```

Comment:
WriteCheckIn might be a slightly clearer API name i think? since ID could be referring to some other ID - or we could just create a separate folder for checkin API's